ner_processing/master_mapping.py

```diff
@@ -63,231 +24,537 @@
-    18: "Module A Temperature",
-    19: "Module B Temperature",
-    20: "Module C Temperature",
-    21: "Gate Driver Board Temperature",
-    22: "Control Board Temperature",
-    23: "RTD #1 Temperature",
-    24: "RTD #2 Temperature",
-    25: "RTD #3 Temperature",
-    26: "RTD #4 Temperature",
-    27: "RTD #5 Temperature",
-    28: "Motor Temperature",
-    29: "Torque Shudder",
-    30: "Analog Input 1",
-    31: "Analog Input 2",
-    32: "Analog Input 3",
-    33: "Analog Input 4",
-    34: "Analog Input 5",
-    35: "Analog Input 6",
-    36: "Digital Input 1",
-    37: "Digital Input 2",
-    38: "Digital Input 3",
-    39: "Digital Input 4",
-    40: "Digital Input 5",
-    41: "Digital Input 6",
-    42: "Digital Input 7",
-    43: "Digital Input 8",
-    44: "Motor Angle (Electrical)",
-    45: "Motor Speed",
-    46: "Electrical Output Frequency",
-    47: "Delta Resolver Filtered", 
-    48: "Phase A Current",
-    49: "Phase B Current",
-    50: "Phase C Current",
-    51: "DC Bus Current",
-    52: "DC Bus Voltage",
-    53: "Output Voltage",
-    54: "VAB_Vd Voltage",
-    55: "VBC_Vq Voltage",
-    56: "Flux Command",
-    57: "Flux Feedback",
-    58: "Id Feedback",
-    59: "Iq Feedback",
-    60: "1.5V Reference Voltage",
-    61: "2.5V Reference Voltage",
-    62: "5.0V Reference Voltage",
-    63: "12V System Voltage",
-    64: "VSM State",
-    65: "Inverter State",
-    66: "Relay State",
-    67: "Inverter Run Mode",
-    68: "Inverter Active Discharge State",
-    69: "Inverter Command Mode",
-    70: "Inverter Enable State",
-    71: "Inverter Enable Lockout",
-    72: "Direction Command",
-    73: "BMS Active",
-    74: "BMS Limiting Torque",
-    75: "POST Fault Lo",
-    76: "POST Fault Hi",
-    77: "Run Fault Lo",
-    78: "Run Fault Hi",
-    79: "Commanded Torque",
-    80: "Torque Feedback",
-    81: "Power on Timer",
-    82: "Torque Command",
-    83: "Speed Command",
-    84: "Direction Command",
-    85: "Inverter Enable",
-    86: "Inverter Discharge",
-    87: "Speed Mode Enable",
-    88: "Commanded Torque Limit",
-    89: "Pack DCL",
-    90: "Pack CCL",
-    91: "TCU X-Axis Acceleration",
-    92: "TCU Y-Axis Acceleration",
-    93: "TCU Z-Axis Acceleration",
-    94: "TCU Temperature C",
-    95: "TCU Temperature F",
-    96: "Relative Humidity",
-    97: "Cell Voltage Info",
-    98: "GLV Current",
-    99: "Strain Gauge Voltage 1",
-    100: "Strain Gauge Voltage 2",
-    101: "Vehicle Speed (mph)"
+    0: {
+        "name": "Mock Data",
+        "units": "",
+    },
+    1: {
+        "name": "Pack Inst Voltage",
+        "units": "FIND",
+    },
+    2: {
+        "name": "Pack Current",
+        "units": "FIND",
+    },
+    3: {
+        "name": "Pack Amphours",
+        "units": "FIND",
+    },
+    4: {
+        "name": "Pack SOC",
+        "units": "FIND",
+    },
+    5: {
+        "name": "Pack Health",
+        "units": "FIND",
+    },
+    6: {
+        "name": "Pack Current",
```

Comment:
This was supposed to be "Failsafe Statuses"
```suggestion
        "name": "Failsafe Statuses",
```